docs/source/index.rst

```diff
@@ -189,6 +189,21 @@ The following information is recorded in the models file:
     references for the model. :samp:`x-backrefs` is not a public interface and
     should not be relied upon as it is subject to change.
 
+It is worth noting that the auto-generated models cannot be imported directly.
+The generated models are added to the `open_alchemy.models` package by the
```

Comment:
In `*.rst` files on the read the docs using \` marks just makes the text italics. To make it look like code I have used ``:samp:`true` ``.

Reply:
Ah makes sense. The double ticks ` `` ` is a trick to make it look like code, which is what I had in mind, but I'll use :samp: for consistency 👍